src/main/js/controls/Line/Line.js

```diff
@@ -184,6 +188,51 @@ class Line extends GraphContent {
         return this;
     }
 
+    /**
+     * @inheritdoc
+     */
+    reflow(graph, graphData) {
+        this.config.values = graphData.values; // Update the old Line config values with the new provided values
```

Comment:
do we need to clear the old line config values before updating it everytime ? is there a scenario where graphData.values can be empty ?

Reply:
>do we need to clear the old line config values before updating it everytime ?

Yes.

>is there a scenario where graphData.values can be empty ?

Yes, graphData.values can be empty. In this scenario, if the user wants to remove all the data associated with a specific key, they can send values to be an empty array. If the key is present all the data is removed else graph remains as it is and only the shift takes place.

Note: will keep a check for the graphData.values being null in a further commit.